corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -1007,6 +1008,42 @@ hqDefine('geospatial/js/models', [
             utils.downloadCsv(casesToExport, headers, cols, 'Case Assignment Export');
         };
 
+        self.acceptAssignments = function () {
+            let caseIdToOwnerId = {};
+            for (const caseItem of self.mapModel.caseMapItems()) {
+                const caseData = self.mapModel.caseGroupsIndex[caseItem.itemId];
+                if ('assignedUserId' in caseData && caseData.assignedUserId) {
+                    caseIdToOwnerId[caseData.item.itemId] = caseData.assignedUserId;
+                }
+            }
+            let requestData = {
+                'case_id_to_owner_id': caseIdToOwnerId,
+                'include_related_cases': self.includeRelatedCases(),
+            };
+
+            $.ajax({
+                type: 'post',
+                url: reassignCasesUrl,
+                dataType: 'json',
+                data: JSON.stringify(requestData),
+                contentType: "application/json; charset=utf-8",
+                success: function (response) {
+                    if (!response.success) {
+                        return alertUser.alert_user(response.message, 'danger');
+                    }
+                    alertUser.alert_user(response.message, 'success', false, true);
```

Comment:
nit: is there a way to be explicit here about what is `false` and `true` for?

Reply:
Having named parameters would be nice , I am afraid there is no direct way to do this.